src/Components/IndividualArticle.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getIndividualArticle } from "../api";
+import formatDate from "../utils/utils";
+import ArticleInfoBar from "./ArticleInfoBar";
+
+const IndividualArticle = () => {
+  const {article_id} = useParams();
+  const [article, setArticle] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    getIndividualArticle(article_id)
+      .then((articleRes) => {
+        setArticle(articleRes);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setError(err);
+        setIsLoading(false);
+      });
+  }, [article_id]);
+
+  if (isLoading) {
```

Comment:
This loading state provides nice user feedback when waiting for the data to be returned from the api :)